docs/architecture/blazor-for-web-forms-developers/state-management.md

```diff
@@ -1,22 +1,98 @@
 ---
 title: State management
 description: Learn about different approaches for managing state in ASP.NET Web Forms and Blazor.
-author: danroth27
-ms.author: daroth
-ms.date: 09/11/2019
+author: csharpfritz
+ms.author: jefritz
+ms.date: 15/04/2020
```

Comment:
```suggestion
ms.date: 05/15/2020
```

The date formatting got you again, I wrote a cool feature in the Docs Authoring Pack - which adds a right-click content menu for updating metadata within a markdown file. If you're using Visual Studio Code, and have the Docs Authoring Pack extension installed try it out.

https://docs.microsoft.com/en-us/contribute/docs-authoring/metadata-updates#update-msdate-metadata-value

![image](https://user-images.githubusercontent.com/7679720/82055163-b897e300-9685-11ea-8abe-8bf229c1fa31.png)